Services/Common/OJS.Services.Common.Data/Implementations/DataService.cs

```diff
@@ -152,5 +151,22 @@ public virtual IQueryable<TEntity> GetByIdQuery(object id)
 
             return query;
         }
+
+        public async Task ExecuteSqlCommandWithTimeout(string query, int timeoutInSeconds)
+        {
```

Comment:
Should be added also to the interface. Add description what means to be executed with timeoutInSeconds = 0